tree/ntuple/v7/test/rfield_vector.cxx

```diff
@@ -83,7 +83,7 @@ TEST(RNTuple, InsideCollection)
    field->SetOnDiskId(idKlassVec);
    field->ConnectPageSource(*source);
 
-   auto fieldCardinality = RFieldBase::Create("", "ROOT::Experimental::RNTupleCardinality").Unwrap();
+   auto fieldCardinality = RFieldBase::Create("", "ROOT::Experimental::RNTupleCardinality<std::uint64_t>").Unwrap();
```

Comment:
It seems that there is no test for `ROOT::Experimental::RNTupleCardinality<std::uint32_t>` here (although it is tested as part of the `RNTupleImporter`).  I guess we should add one (?)